eva_sub_cli/semantic_metadata.py

```diff
@@ -74,6 +75,18 @@ def check_all_taxonomy_codes(self):
                 self.check_taxonomy_code(sample[BIOSAMPLE_OBJECT_KEY][CHARACTERISTICS_KEY][TAX_ID_KEY][0]['text'],
                                          f'/{SAMPLE_KEY}/{idx}/{BIOSAMPLE_OBJECT_KEY}/{CHARACTERISTICS_KEY}/{TAX_ID_KEY}')
 
+    def check_uniqueness_analysis_alias(self):
+        analyses = self.metadata[ANALYSIS_KEY]
+        analysis_aliases = [analysis.get(ANALYSIS_ALIAS_KEY) for analysis in analyses]
+        if len(analysis_aliases) != len(set(analysis_aliases)):
+            counter = Counter(analysis_aliases)
+            for idx, analysis in enumerate(analyses):
+                if counter[analysis.get(ANALYSIS_ALIAS_KEY)] > 1:
+                    self.add_error(
+                        f'/{ANALYSIS_KEY}/{idx}/{ANALYSIS_ALIAS_KEY}',
+                        f'Analysis alias {analysis.get(ANALYSIS_ALIAS_KEY)} is present '
+                        f'{counter.get(counter[analysis.get(ANALYSIS_ALIAS_KEY)])} times in the Analysis Sheet')
```

Comment:
```suggestion
                        f'{counter[analysis.get(ANALYSIS_ALIAS_KEY)]} times in the Analysis Sheet')
```
Should fix the test as well.